moveit_ros/hybrid_planning/test/config/demo_controller.yaml

```diff
@@ -5,21 +5,14 @@ controller_manager:
       type: joint_trajectory_controller/JointTrajectoryController
     joint_state_controller:
       type: joint_state_controller/JointStateController
-    panda_joint_group_position_controller:
```

Comment:
@AdamPettinger Why did you remove this controller config?

Reply:
Ah, sorry I missed that point. I was copying the servo and moveit_cpp demos since an update to ros2_control launching (looks like it happened after you started this work), see [the current servo launch files](https://github.com/ros-planning/moveit2/blob/9ca817dfc19b70509d50d367625f7f55521b2b4e/moveit_ros/moveit_servo/launch/servo_teleop.launch.py#L69-L94). I would be more than happy to put it back and have both interfaces available